partcad/src/partcad/wrappers/wrapper_render_raster.py

```
@@ -0,0 +1,112 @@
+        raise ValueError("Invalid background color: %r" % (value,))
+    return int(text, 16)
+
+
+def process(path, request, fmt, config_pil=None):
+    """Render the requested shape into 'path' as a raster image.
+
+    'fmt' is the format name renderPM expects ("PNG", "JPEG"); 'config_pil' is
+    passed through to 'PIL.Image.save()' by renderPM.
+    """
+    try:
+        svg_path = tempfile.mktemp(".svg")
```

Ruff (0.16.1)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)